unity-game/Assets/Scripts/ElevatorScripts/ElevatorBookends.cs

```diff
@@ -0,0 +1,24 @@
+﻿using UnityEngine;
+using System.Collections;
+
+public class ElevatorBookends : MonoBehaviour {
```

Comment:
this class is probably unnecessary, these references could be put into a base moving platform class